docker/Dockerfile

```
@@ -0,0 +1,13 @@
+ARG swift_version=5.6
+ARG ubuntu_version=bionic
```

We should use `focal` here
```suggestion
ARG ubuntu_version=focal
```